family_context/settings.py

```diff
@@ -38,6 +39,48 @@
     default="https://127.0.0.1",
 )
 
+LOG_LEVEL = config("LOG_LEVEL", default="INFO")
+LOG_DESTINATION = config("LOG_DESTINATION", default="file_output")
+choose_log_handler = {"file_output": "file", "console_output": "console"}
```

Comment:
If `LOG_DESTINATION` is just "file" or "console" you can remove the `choose_log_handler` dict, no?

Reply:
On one hand I can.
On the other hand, I was trying to avoid tightly coupling external variable names and internal variable names. So that external variable names can always be made to be more memorable/user-friendly while internal variable names are guaranteed to stay the same. 
So choose_log_handler is the isolation layer.

Then again, it is not a necessary feature now so it can be removed.